static/app/views/starfish/queries/useFullSpanFromTrace.tsx

```diff
@@ -9,14 +9,19 @@ import {SpanIndexedField} from 'sentry/views/starfish/types';
 export function useFullSpanFromTrace(
   group?: string,
   sorts?: Sort[],
-  enabled: boolean = true
+  enabled: boolean = true,
+  extraFilters?: Record<string, string>
 ) {
-  const filters: {[key: string]: string} = {};
+  let filters: {[key: string]: string} = {};
```

Comment:
```suggestion
  const filters: {[key: string]: string} = { ...extraFilters };
```

Shouldn't `SpanIndexedField.SPAN_GROUP` always override pass in extra filters?

Reply:
I'm sure why you would intentionally add a spanGroup to any extra filters, but yeah i think that makes sense, i'll update it